install/installer.class.php

```diff
@@ -1377,14 +1377,146 @@ static public function getPolicies() {
          ],
 
          [
-            'name'                                => __('Disable all sounds', 'flyvemdm'),
-            'symbol'                              => 'disableAllSounds',
+            'name'                                => __('Disable media sounds', 'flyvemdm'),
+            'symbol'                              => 'disableStreamMusic',
             'group'                               => 'ui',
             'type'                                => 'bool',
             'type_data'                           => '',
             'unicity'                             => 1,
             'plugin_flyvemdm_policycategories_id' => 'Security > User interface',
-            'comment'                             => __('Disable all sounds of device', 'flyvemdm'),
+            'comment'                             => __('Disable all media sounds from device', 'flyvemdm'),
+            'default_value'                       => '0',
+            'recommended_value'                   => '0',
+            'is_android_policy'                   => '1',
+            'is_android_system'                   => '1',
+            'is_apple_policy'                     => '0',
+         ],
+
+         [
+            'name'                                => __('Disable ringer sounds', 'flyvemdm'),
+            'symbol'                              => 'disableStreamRing',
+            'group'                               => 'ui',
+            'type'                                => 'bool',
+            'type_data'                           => '',
+            'unicity'                             => 1,
+            'plugin_flyvemdm_policycategories_id' => 'Security > User interface',
```

Comment:
Please, change category of these policies to "User interface" (without beinig a child of Security). We need to check the install and upgrade code handles correctly with 2 categories having the same short name.